src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PhysicalDetailResourceTest.kt

```diff
@@ -217,41 +267,38 @@ class PhysicalDetailResourceTest : QueueIntegrationTest() {
     )
 
     @Test
-    fun `find by cell location with prison prefix`(): Unit = physicalDetailSearch(
```

Comment:
I guess this and the next test were accidentally left here in `height and weight tests`? They definitely still exist in their proper home don't they?